stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -41,47 +41,42 @@ namespace {
 
 Attribute convertAttrToStablehlo(Attribute vhloAttr) {
   LLVM_DEBUG(llvm::dbgs() << "Converting " << vhloAttr);
-  if (auto attr = vhloAttr.dyn_cast<vhlo::ChannelHandleAttr>()) {
+  if (auto attr = vhloAttr.dyn_cast<vhlo::ChannelHandleAttr>())
     return stablehlo::ChannelHandleAttr::get(attr.getContext(),
                                              attr.getHandle(), attr.getType());
-  }
   if (auto attr = vhloAttr.dyn_cast<vhlo::ComparisonDirectionAttr>()) {
     RETURN_CONVERTED_ENUM_ATTR(ComparisonDirection);
   }
   if (auto attr = vhloAttr.dyn_cast<vhlo::ComparisonTypeAttr>()) {
     RETURN_CONVERTED_ENUM_ATTR(ComparisonType);
   }
-  if (auto attr = vhloAttr.dyn_cast<vhlo::ConvDimensionNumbersAttr>()) {
+  if (auto attr = vhloAttr.dyn_cast<vhlo::ConvDimensionNumbersAttr>())
     return stablehlo::ConvDimensionNumbersAttr::get(
         attr.getContext(), attr.getInputBatchDimension(),
         attr.getInputFeatureDimension(), attr.getInputSpatialDimensions(),
         attr.getKernelInputFeatureDimension(),
         attr.getKernelOutputFeatureDimension(),
         attr.getKernelSpatialDimensions(), attr.getOutputBatchDimension(),
         attr.getOutputFeatureDimension(), attr.getOutputSpatialDimensions());
-  }
   if (auto attr = vhloAttr.dyn_cast<vhlo::CustomCallApiVersionAttr>()) {
     RETURN_CONVERTED_ENUM_ATTR(CustomCallApiVersion);
   }
-  if (auto attr = vhloAttr.dyn_cast<vhlo::DotDimensionNumbersAttr>()) {
+  if (auto attr = vhloAttr.dyn_cast<vhlo::DotDimensionNumbersAttr>())
     return stablehlo::DotDimensionNumbersAttr::get(
         attr.getContext(), attr.getLhsBatchingDimensions(),
         attr.getRhsBatchingDimensions(), attr.getLhsContractingDimensions(),
         attr.getRhsContractingDimensions());
-  }
   if (auto attr = vhloAttr.dyn_cast<vhlo::FftTypeAttr>()) {
     RETURN_CONVERTED_ENUM_ATTR(FftType);
   }
```

Comment:
Still not sure what's the right thing to do for a chain of if's where some have braces and others do not. This feels strange with a mix.

Reply:
Since this one is not clear, let's just leave this file unchanged for consistency in the file. Wdyt?